README.md

```diff
@@ -828,6 +932,15 @@ Operands:
 Kind: *Instruction*
 
 Errors: *Fallible*.
+- `BadOrigin`
+- `ReanchorFailed`
+- `AssetNotFound`
+- `Unroutable`
+- `DestinationUnsupported`
+- `ExceedsMaxMessageSize`
+- `Transport`
+- `NotHoldingFees`
+- `FailedToTransactAsset`
```

Comment:
Missing `LockError` here and on `UnlockAsset`, `NoteUnlockable`, `RequestUnlock`